activestate.yaml

```diff
@@ -68,6 +68,9 @@ scripts:
       export GO111MODULE=on
       goflags="${GOFLAGS}"; unset GOFLAGS
       GOFLAGS="${goflags}"
+      pushd ~
+      go install github.com/sanderhahn/gozip/cmd/gozip@latest
+      popd
```

Comment:
1. This should be vendored (or at least not using latest) so the build is reproducible
2. You should be using our fork of gozip from here: https://github.com/ActiveState/gozip

Reply:
Why do we have a fork? We've made no changes to it.